src/core/MOM_open_boundary.F90

```diff
@@ -4648,6 +4744,123 @@ subroutine register_temp_salt_segments(GV, US, OBC, tr_Reg, param_file)
 
 end subroutine register_temp_salt_segments
 
+!> Sets the OBC properties of external obgc tracers, such as their source file and field name
+subroutine set_obgc_segments_props(tr_name,obc_src_file_name,obc_src_field_name,lfac_in,lfac_out)
+  character(len=*),  intent(in) :: tr_name            !< Tracer name
+  character(len=*),  intent(in) :: obc_src_file_name  !< OBC source file name
+  character(len=*),  intent(in) :: obc_src_field_name !< name of the field in the source file
+  real,              intent(in) :: lfac_in            !< factors for tracer reservoir length scales
+  real,              intent(in) :: lfac_out           !< factors for tracer reservoir length scales
+
+  type(external_tracers_segments_props),pointer :: node_ptr => NULL() !pointer to type that keeps
+                                                                    ! the tracer segment properties
+  allocate(node_ptr)
+  node_ptr%tracer_name = trim(tr_name)
+  node_ptr%tracer_src_file = trim(obc_src_file_name)
+  node_ptr%tracer_src_field = trim(obc_src_field_name)
+  node_ptr%lfac_in  = lfac_in
+  node_ptr%lfac_out = lfac_out
+  !Reversed Linked List implementation! Make this new node to be the head of the list.
+  node_ptr%next => obgc_segments_props
+  obgc_segments_props => node_ptr
+  num_obgc_tracers = num_obgc_tracers+1
+end subroutine set_obgc_segments_props
+
+!> Get the OBC properties of external obgc tracers, such as their source file, field name,
+!  reservoir length scale factors
```

Comment:
The continuation line of this comment should start with `!!` to use the right doxygen syntax.